tests/test_cases/test_vhdl_libraries/Makefile

```diff
@@ -0,0 +1,14 @@
+SIM ?= ghdl
+TOPLEVEL_LANG ?= vhdl
+VHDL_SOURCES_blib := b.vhdl
+VHDL_SOURCES := a.vhdl
+TOPLEVEL := a
+MODULE := test_ab
+
+ifneq ($(shell echo $(TOPLEVEL_LANG) | tr A-Z a-z),vhdl)
```

Comment:
One needs to restrict this also to ghdl/modesim/questa) since Riviera will fail. (Or add this functionality to Riviera)
Example: https://github.com/cocotb/cocotb/blob/aa956f83e119e4ca4615c5b8660eb15c4dfa8623/examples/mixed_signal/tests/Makefile#L67

Reply:
Yeah, it's only supported in GHDL/ModelSim/Questa for now. @ktbarrett mentioned adding fix, so I won't interfere for a bit.